composer.json

```diff
@@ -17,6 +17,11 @@
         "drush/drush": "^10.3"
     },
     "config": {
+        "allow-plugins": {
+            "composer/installers": true,
+            "cweagans/composer-patches": true,
+            "drupal/core-composer-scaffold": true
+        },
         "sort-packages": true
```

Comment:
I want `composer install` locally to get the prompts based off of the current `composer.lock`